src/Yandex.Alice.Sdk/Models/AliceEntityDateTimeValueModel.cs

```diff
@@ -1,15 +1,42 @@
 ﻿namespace Yandex.Alice.Sdk.Models
 {
+    using System;
     using System.Text.Json.Serialization;
     using JetBrains.Annotations;
 
     [UsedImplicitly(ImplicitUseTargetFlags.WithMembers)]
     public class AliceEntityDateTimeValueModel
     {
+        [JsonPropertyName("year")]
+        public int Year { get; set; }
+
+        [JsonPropertyName("year_is_relative")]
+        public bool YearIsRelative { get; set; }
+
+        [JsonPropertyName("month")]
+        public int Month { get; set; }
+
+        [JsonPropertyName("month_is_relative")]
+        public bool MonthIsRelative { get; set; }
+
         [JsonPropertyName("day")]
-        public double Day { get; set; }
+        public int Day { get; set; }
 
         [JsonPropertyName("day_is_relative")]
         public bool DayIsRelative { get; set; }
+
+        [JsonPropertyName("hour")]
+        public int Hour { get; set; }
+
+        [JsonPropertyName("hour_is_relative")]
+        public bool HourIsRelative { get; set; }
+
+        [JsonPropertyName("minute")]
+        public int Minute { get; set; }
+
+        [JsonPropertyName("minute_is_relative")]
+        public bool MinuteIsRelative { get; set; }
+
+        public DateTime DateTime => new DateTime(Year, Month, Day, Hour, Minute, 0);
```

Comment:
I think we should delete or move this property out of this class since it's not part of the model. Also, it'll throw an exception in case any of the parameters are relative. For example  `day = -1`.

If we want to keep it then It can be some `GetDate()` extension method of `AliceEntityDateTimeValueModel` instead. And it should correctly handle relative values. Would be nice to cover that with tests too.

Reply:
Nevermind, I'll remove it